### ISSUE TITLE
Plugin Install Failed

### ISSUE THREAD

**Issue by DaDD65** [dgreif/ring#548]
# Bug Report

### Describe the Bug
After updating to 9.14.1 cameras stop responding in Apple Home.  Uninstalled plugin.  Attempted to re-install, plugin installed failed. Log states:  Invalid response body while trying to fetch https://registry.npmjs.org/@types%2fnode: Integrity verification failed for sha512-gwEYQdQ4e0VlMmJ54vMxro8Gt5fCYSzCSSBPuAtko7NJOTAVlDFnFfuZo2d3AvBUJTnx5PNWYt6sOJRh017SSg== (/home/hoobs/.npm/_cacache/content-v2/sha512/83/01/1841d4387b4565326279e2f331ae8f06b797c2612cc249204fb80b64a3b34939301594316715fb99a3677702f0542539f1e4f35662deac389461d35ed24a)

### To Reproduce
 - Start Homebridge
 - Attempt to install
 - Failed: Invalid response body while trying to fetch https://registry.npmjs.org/@types%2fnode: Integrity verification failed for sha512-gwEYQdQ4e0VlMmJ54vMxro8Gt5fCYSzCSSBPuAtko7NJOTAVlDFnFfuZo2d3AvBUJTnx5PNWYt6sOJRh017SSg== (/home/hoobs/.npm/_cacache/content-v2/sha512/83/01/1841d4387b4565326279e2f331ae8f06b797c2612cc249204fb80b64a3b34939301594316715fb99a3677702f0542539f1e4f35662deac389461d35ed24a)

### Expected behavior
 plugin installs

### Screenshots/Logs
 If applicable, add screenshots and log output to help explain your problem.

### Additional context
 Add any other context about the problem here.

### Homebridge Ring Config
Post homebridge-ring platform config  _without sensitive information_
```

HIGHLIGHT THIS TEXT AND POST CONFIG HERE
DO NOT INCLUDE YOUR REFRESH TOKEN OR EMAIL/PASSWORD (replace them with xxx)

```

### Environment

 - OS: Raspbian 10
 - Node.js: 12.19.0
 - NPM: [e.g 6.13.4] 
 - homebridge-ring: [e.g 6.0.3]
 - homebridge: [e.g. 0.4.50 or N/A]
 - hoobs: 3.3.2

**Comment by dgreif** [comment]
This is an issue with your local npm, rather than the plugin.  You should be able to run `npm cache verify` and then try the install again.  If that doesn't work, run `npm cache clean --force` and then try the install again

**Comment by DaDD65** [comment]
Thanks for the help.  New at this only had the box for a short time.  npm cache verify solved the problem.  Was able to install the the plugin successfully.  Added all my Ring devices to Homekit.  All Ring devices show as offline.  Hoobs log running constantly with"[Ring] Failed to reach Ring server at https://api.ring.com/clients_api/ring_devices. Stream closed with error code NGHTTP2_ENHANCE_YOUR_CALM. Trying again in 5 seconds."

**Comment by dgreif** [comment]
@DaDD65 that error has been intermittent but I thought it was resolved.  Make sure you are on the latest version of the plugin and consider updating to node 14

**Comment by DaDD65** [comment]
Plugin is 9.14.1 Node is 12.19.0 HOOBS 3.3.2  Don't see how to update the node?

**Comment by searay345** [comment]
I’m experiencing the same issue as @DaDD65. I did not uninstall the ring plugin. But looking at the logs, I’m getting the same string. 
Failed to reach Ring server at https://api.ring.com/clients_api/ring_devices. Stream closed with error code NGHTTP2_ENHANCE_YOUR_CALM. Trying again in 5 seconds."
This has been going on for many hours now.

**Comment by bjbires1248** [comment]
I am having the same issue as @searay345

**Comment by dgreif** [comment]
Try updating to the latest version of NodeJS.  On HOOBS, you can run `wget -q -O - http://bit.ly/get-hoobs | sudo bash /dev/stdin --node lts`

**Comment by DaDD65** [comment]
Thanks.  I researched that yesterday.  Didn't try it yet.  Saw that sometimes getting back in was problematic after a node update.
Solution was to do this from Windows via Putty.
Sometimes after node is updated the system needs rebuilding (via SSH as the web interface won't be available). The commands are (to be run separately)
sudo npm uninstall -g @hoobs/hoobs
sudo npm install -g --unsafe-perm @hoobs/hoobs
Do you think this is the cause of the Ring error or is it a Ring server issue?

**Comment by dgreif** [comment]
@DaDD65 the `NGHTTP2_ENHANCE_YOUR_CALM` is happening because some versions of node don't handle http2 properly.  If you update node, that error should go away.  If you need help updating node (beyond the command I provided), I suggest you reach out to HOOBS support

**Comment by searay345** [comment]
@dgreif Thank you! It appears updating to node 14.15.1 solved the issue. 

I took a backup of my system before running the command. And I also used an SSH client on my phone to get into the CLI rather than use the HOOB's console on the website. It worked great and only took ~3-4 minutes to complete. Once it completed, I did a sudo reboot and watched my stuff come back to life in the Home App. RING Cameras starting working right away (snapshots & live views).
